cmd/version.go

```diff
@@ -3,24 +3,25 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/operate-first/opfcli/api"
 	"github.com/operate-first/opfcli/version"
 	"github.com/spf13/cobra"
 )
 
-var versionCmd = &cobra.Command{
-	Use:           "version",
-	Short:         "Show opfcli version information",
-	SilenceUsage:  true,
-	SilenceErrors: true,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		fmt.Printf("Name: %s\n", version.Name)
-		fmt.Printf("Version: %s\n", version.BuildVersion)
-		fmt.Printf("Git Commit Hash: %s\n", version.BuildHash)
-		fmt.Printf("Build Date: %s\n", version.BuildDate)
-		return nil
-	},
-}
+func NewCmdVersion(api *api.API) *cobra.Command {
```

Comment:
this function doesn't seem like it uses `api`

Reply:
It's doesn't (right now), but this is effectively the standard function signature for new subcommands. I'm inclined to leave this as-is.